src/api/videoRoom.ts

```diff
@@ -0,0 +1,79 @@
+import Bugsnag from "@bugsnag/js";
+import firebase from "firebase/app";
+
+import { VideoChatRequest, VideoChatRequestState } from "types/VideoRoom";
+
+export const inviteToVideoChat = async (
+  hostUserId: string,
+  hostUserLocation: string,
+  invitedUserId: string
+) => {
+  const videoRoomRequest: VideoChatRequest = {
+    hostUserId: hostUserId,
+    hostUserLocation: hostUserLocation,
+    invitedUserId: invitedUserId,
+    invitedUserLocation: "",
+    state: VideoChatRequestState.Invited,
+    createdAt: firebase.firestore.FieldValue.serverTimestamp(),
```

Comment:
This is where I said to override the type. We could either be hacky and use `as` (not ideal), or we could use TypeScript 'utility types' to create a new type like `CreateVideoChatRequest`. 

I don't believe there is a standard utility type for changing the type of a key, but I found this:

- https://dev.to/denniscual/typescript-hack-simple-utility-type-for-changing-type-of-keys-4bba

It would be good to add that to our own `types/utility` file, then use it as follows.

This should be done in the appropriate `types/*` file, something like:

```typescript
export type CreateVideoChatRequest = ChangeTypeOfKeys<VideoChatRequest, 'createdAt', FieldValue>
```

Reply:
Also should probably be `CreateVideoRoomRequest` or even `CreateVideoRoomInvite` or similar maybe?